frontend/index.html

```diff
@@ -117,46 +117,55 @@ <h1 class="intro-title">Improve your digital storytelling with a combination of
     <div class="carousel-section-wrapper">
       <h2 class="section-label">An award winning platform</h2>
       <div class="owl-carousel owl-theme awards-carousel">
-        <div class="item">
-          <img src="images/award1.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
-        </div>
-        <div class="item">
-          <img src="images/award2.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
-        </div>
-        <div class="item">
-          <img src="images/award3.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
-        </div>
-        <div class="item">
-          <img src="images/award1.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
-        </div>
-        <div class="item">
-          <img src="images/award3.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
-        </div>
-        <div class="item">
-          <img src="images/award2.png" alt="">
-          <p>
-            European Commission <br> Digital Skills Award 2016
-          </p>
+          <div class="item">
+              <img src="images/hha.gif" alt="">
+              <p>
+                UK Digital Experience Award <br> Customer Experience Innovation
```

Comment:
Awesome work here @andela-iakande, we need to change the text below the images to the name of the respective partners.

Reply:
Let's also add these changes to coursebuilder/modules/explorer/_static/components/pages/homepage.html